sdk/storage/storage-queue/src/utils/utils.common.ts

```diff
@@ -134,6 +158,12 @@ export function extractConnectionStringParts(
     return "";
   }
 
+  if (connectionString.search("UseDevelopmentStorage=true") !== -1) {
```

Comment:
nit: should it be `connectionString.startsWith`?

Reply:
Pay attention to `search` is required higher Node.js version than v8, or whether it needs additional polyfill in IE. If so, update polyfill list in readme.md

Reply:
Updated this instance to `connectionString.startsWith`.

@XiaoningLiu, there are other instances of `.search` that got checked-in for the previous previews. The tests didn't have any issue with IE. 
And we support Node V8 or higher only.
So, we don't have to care about anything else?